src/main/resources/application.properties

```diff
@@ -8,13 +8,23 @@ spring.jpa.hibernate.ddl-auto=update
 spring.liquibase.change-log=classpath:db/liq/changelog.xml
 management.endpoint.shutdown.enabled=true
 management.endpoints.web.exposure.include=health,shutdown
+
 ## Keycloak configuration ##
 # keycloak.enabled=false
-keycloak.authServerUrl=${KEYCLOAK_URL}
-keycloak.realm=${KEYCLOAK_REALM}
-keycloak.clientId=${KEYCLOAK_ID}
-keycloak.clientSecret=${KEYCLOAK_SECRET}
-keycloak.tokenUri=${keycloak.authServerUrl}/realms/app/protocol/openid-connect/token
+keycloak.authServerUrl=http://16.16.197.13:8082/
+keycloak.realm=app
+keycloak.clientId=app-client
+keycloak.clientSecret=T4ZEwkGDl1iQRAKLbXvQ4uvwkc2tK8ke
+keycloak.tokenUri=http://16.16.197.13:8082/realms/app/protocol/openid-connect/token
+keycloak.logoutUri=http://16.16.197.13:8082/realms/app/protocol/openid-connect/logout
+spring.security.oauth2.jwt.jwk-set-uri=http://16.16.197.13:8082/realms/app/protocol/openid-connect/certs
+
+spring.security.oauth2.resourceserver.jwt.issuer-uri=http://16.16.197.13:8082/auth/realms/app
```

Comment:
If I am not mistaken we just need
`spring.security.oauth2.resourceserver.jwt.jwk-set-uri=${keycloak.authServerUrl}/realms/app/protocol/openid-connect/certs`

I don't think we need spring.security.oauth2.jwt.jwk-set-uri and 
 spring.security.oauth2.resourceserver.jwt.issuer-uri